aiohttp/connector.py

```diff
@@ -573,6 +574,13 @@ def _resolve_host(self, host, port):
                 host, port, family=self._family)
             return res
 
+    def _host_is_ip(self, host):
+        try:
+            ipaddress.ip_address(host)
+            return True
+        except ValueError:
+            return False
```

Comment:
Everybody most of their time use host names not their IP addresses,
so most of time this method will raise and except ValueError what is not very efficient.